recipes/default.rb

```diff
@@ -7,6 +7,7 @@
 end
 
 chef_gem 'influxdb' do
+  version '0.6.1'
```

Comment:
This change requires a newer version of the influxdb ruby client now because the new version exposes support for setting the shard duration on a retention policy.